libs/ui/packages/native/src/components/Layout/List/VerticalStepper/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import StepperItem from "./StepperItem";
+import { Flex } from "../..";
+import { BaseStyledProps } from "src/components/styled";
+import { Item, ItemStatus } from "../types";
+
+export type Props = BaseStyledProps & {
+  steps?: Item[];
+  setActiveIndex?: (arg0: number) => void;
```

Comment:
very optional: I'd name the arg "index" or "newIndex" here to be more explicit

Reply:
It's stuff for debugging @ofreyssinet-ledger added in the VerticalTimeline at some point, I think it's better to keep the two consistent